src/sentry/api/endpoints/project_rule_details.py

```diff
@@ -285,7 +291,13 @@ def put(self, request: Request, project, rule) -> Response:
                 context = {"uuid": client.uuid}
                 return Response(context, status=202)
 
-            trigger_sentry_app_action_creators_for_issues(actions=kwargs["actions"])
+            try:
+                trigger_sentry_app_action_creators_for_issues(actions=kwargs["actions"])
+            except (SentryAppError, SentryAppIntegratorError, SentryAppSentryError) as e:
+                response: dict[str, Any] = {"actions": [e.message]}
```

Comment:
"actions" is the key because our frontend looks for specific words for some reason